packages/infra/lib/api-stack.ts

```diff
@@ -299,6 +300,20 @@ export class APIStack extends Stack {
       sentryDsn: props.config.lambdasSentryDSN,
     });
 
+    const fhirToMedicalRecordLambda = this.setupFhirToMedicalRecordLambda({
+      lambdaLayers,
+      vpc: this.vpc,
+      convertDocLambdaName: cdaToVisualizationLambda.functionName,
+      dynamoDBSidechainKeysTable,
+      converterUrl: props.config.fhirToCDAUrl,
+      bucketName: isSandbox(props.config)
+        ? props.config.sandboxSeedDataBucketName
+        : medicalDocumentsBucket.bucketName,
```

Comment:
how's this going to work in sandbox? don't we need to:
- pre-generate some summary CDAs for sandbox patients
- store them on S3
- handle this case in the lambda where we return the link to the existing document in sandbox

Reply:
This has been updated just need to add the xml's i was adding htmls and pdfs and realized thats the wrong approach